secure_uninstall/__openerp__.py

```diff
@@ -0,0 +1,50 @@
+# coding: utf-8
```

Comment:
@moylop260 
in french
http://sametmax.com/un-header-dencoding-plus-simple-pour-python
in english

http://translate.google.com/translate?hl=fr&sl=auto&tl=en&u=http%3A%2F%2Fsametmax.com%2Fun-header-dencoding-plus-simple-pour-python%2F&sandbox=1